src/main/java/de/tum/in/www1/artemis/repository/StudentScoreRepository.java

```diff
@@ -45,6 +45,7 @@
             """)
     Optional<StudentScore> findStudentScoreByExerciseAndUserLazy(@Param("exercise") Exercise exercise, @Param("user") User user);
 
+    // TODO: use a custom object instead of Object[] (as in the example above with ParticipantScoreAverageDTO)
```

Comment:
Note: we should use a record here in future, see e.g. https://www.baeldung.com/hibernate-query-to-custom-class